axon-server-connector/src/main/java/org/axonframework/axonserver/connector/command/AxonServerCommandBus.java

```diff
@@ -276,6 +276,7 @@ public void onNext(CommandProviderInbound commandToSubscriber) {
                     public void onError(Throwable throwable) {
                         logger.warn("Received error from server: {}", throwable.getMessage());
                         subscriberStreamObserver = null;
+                        resubscribe();
```

Comment:
Do not resubscribe when error is StatusRuntimeException with Error code UNAVAILABLE. In this case the AxonServer is gone, which is handled by the PlatformConnectionManager.